src/core/dbus-manager.c

```diff
@@ -2976,6 +2990,16 @@ const sd_bus_vtable bus_manager_vtable[] = {
                                  SD_BUS_PARAM(changes),
                                  method_enable_unit_files,
                                  SD_BUS_VTABLE_UNPRIVILEGED),
+        SD_BUS_METHOD_WITH_NAMES("EnablePortableUnitFiles",
+                                 "asbb",
+                                 SD_BUS_PARAM(files)
+                                 SD_BUS_PARAM(runtime)
+                                 SD_BUS_PARAM(force),
```

Comment:
hmm, I think it would be better to add EnableUnitFilesWithFlags() instead, and then move runtime + force into a uint64_t flags parameter, and then add the portable thing there too. I figure sooner or later we might get more of these, and given how much the D-Bus method extension logic sucks maybe a flags param is the better way out.

Reply:
Ok, done